src/brsTypes/components/BrsObjects.ts

```diff
@@ -39,3 +40,26 @@ export const BrsObjects = new Map<string, Function>([
     ["roint", (_: Interpreter, literal: Int32) => new roInt(literal)],
     ["roinvalid", (_: Interpreter) => new roInvalid()],
 ]);
+
+/**
+ * Lets another software using BRS as a library to add/overwrite an implementation of a BrsObject.
+ *
+ * This is useful, for example, if another piece of software wanted to implement video playback or Draw2d functionality.
+ *
+ * In each element of the objectList param, it is pair:
+ * [name of the BrsObject (e.g. "roScreen", etc.), function (interpreter, ...additionalArgs) that returns a new object]
+ *
+ * @example
+ *
+ * AddAdditionalBrsObjects([
+ *   ["roScreen", (_interpreter) => {return new roScreen();}]
+ * ])
+ *
+ * @export
+ * @param {[string, Function][]} objectList - array of pairs: [name, constructor function]
+ */
+export function AddAdditionalBrsObjects(objectList: [string, () => BrsComponent][]): void {
```

Comment:
Most functions in this project are `camelCase`d, and "additional" is pretty redundant here since we're already "add"ing some :) would you mind changing the name to `addBrsObjects` or `extendBrsObjects` or something similar??

```suggestion
export function addBrsObjects(objectList: [string, () => BrsComponent][]): void {
```